MiniProjectUIKit/Component/RecipeCollectionViewCell.swift

```diff
@@ -0,0 +1,108 @@
+//
+//  RecipeCollectionViewCell.swift
+//  MiniProjectUIKit
+//
+//  Created by Vincent Saranang on 06/12/24.
+//
+
+import UIKit
+
+class RecipeCollectionViewCell: UICollectionViewCell {
```

Comment:
better `final` karena class ini ga di inherit dimanapun

```suggestion
final class RecipeCollectionViewCell: UICollectionViewCell {
```